sagemaker-experiments/tensorflow2-california-housing-regression-experiment/data/README.md

```diff
@@ -0,0 +1 @@
+# Data Folder
```

Comment:
Did you mean to add more to the readme file? Or maybe the file is not needed?

Reply:
I will delete the folder and create it in the code